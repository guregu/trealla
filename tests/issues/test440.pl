:- initialization(main).

main :-
	Y =.. [x,[_|Y]],
	write(Y), nl.

