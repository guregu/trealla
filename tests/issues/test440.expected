x([_4|x([_4|...])])
